src/components/molecules/OcSidebarNavItem/OcSidebarNavItem.vue

```diff
@@ -131,6 +148,39 @@ export default {
 </nav>
 ```
 
+### Collapsed nav items
+```js
+<nav class="oc-background-brand">
+  <oc-list>
+    <oc-sidebar-nav-item target="#" icon="folder" :collapsed="true" active>Files</oc-sidebar-nav-item>
+    <oc-sidebar-nav-item target="#" icon="delete" :collapsed="true">Deleted files</oc-sidebar-nav-item>
+    <oc-sidebar-nav-item target="#" icon="group" :collapsed="true" isolate>Users</oc-sidebar-nav-item>
+  </oc-list>
+</nav>
+```
+
+### Nav item without custom target
+```js
+<template>
+<nav class="oc-background-brand">
+  <oc-list>
+    <oc-sidebar-nav-item>
+      <oc-icon variation="inverse" name="apps" @click="customTarget" />
```

Comment:
Nitpick: It's preferred not to use click-handlers on non-button tags for a11y reasons. The better way to go here would be to use a "raw" on-button component wrapping the icon :)

Reply:
That will also make hover handling better ;-) really not just a nitpick but a hard requirement.

Reply:
ok will replace it ^^

Reply:
done